components/index/VolunteerHero.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="c-hero">
+    <h1 class="c-hero__header1">{{ $t('Volunteer at a Dojo near you') }}</h1>
+    <div class="c-hero__images">
+      <div class="c-hero__image">
+        <img src="https://picsum.photos/480/320?image=1" />
+      </div>
+      <div class="c-hero__image">
+        <img src="https://picsum.photos/480/320?image=2" />
+      </div>
+      <div class="c-hero__image">
+        <img src="https://picsum.photos/480/320?image=3" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+  @import '~/assets/_variables.scss';
+
+  .c-hero {
+    padding: 0 0 80px 0;
+    background-image:
+      radial-gradient(transparent 50%, transparent 100%),
```

Comment:
The radial-gradient isn't needed here. It was just for the curve at the bottom of the gradient on the homepage.